data/formats-data.js

```diff
@@ -3358,7 +3365,7 @@ exports.BattleFormatsData = {
 		tier: "PU",
 	},
 	clamperl: {
-		randomBattleMoves: ["shellsmash", "icebeam", "surf", "hiddenpowergrass", "hiddenpowerelectric", "substitute"],
+		randomBattleMoves: ["shellsmash", "icebeam", "hydropump", "hiddenpowergrass"],
```

Comment:
between it's item, deep sea tooth, and shell smash, surf is the prefered move since you have enough power already (i believe you reach a spa stat around 68?) and reliability is more important for the sweep

Reply:
yeah good call, got stuck in general smasher mentality but forgot the monster this is